### PR TITLE
Adding aguments to Add method.

### PR DESCRIPTION
**Pull request by gogag2**
Delete hard-coded value and pass arguments to process.

ISSUE=#4

### CODE REVIEW

File: examples/calculator_bridge.cc [hunk 1]
```diff
@@ -14,19 +14,57 @@
...
+  assert(status == napi_ok);
+
+  double value1;
+  status = napi_get_value_double(env, args[1], &value1);
+  assert(status == napi_ok);
+
   napi_value sum;
-  napi_status status = napi_create_number(env, Calculator::Add(1, 2), &sum);
-  return status == napi_ok ? sum : nullptr;
+  status = napi_create_number(env, value0 + value1, &sum);
```
Comment (romandev) [inline examples/calculator_bridge.cc]: This is not correct. Your code doesn't use Calculator::Add..

Comment (gogag2) [reply examples/calculator_bridge.cc]: Sure. I will fix it now.

File: examples/calculator_bridge.cc [hunk 2]
```diff
@@ -23,10 +23,32 @@ void Init(napi_env env, napi_value exports, napi_value module, void* priv) {
 }
 
 napi_value Add(napi_env env, napi_callback_info info) {
-  // TODO(zino): Should take arguments and process them.
-  napi_value sum;
-  napi_status status = napi_create_number(env, Calculator::Add(1, 2), &sum);
-  return status == napi_ok ? sum : nullptr;
+  double value0, value1;
```
Comment (romandev) [inline examples/calculator_bridge.cc]: double number1;
double number2;

File: examples/calculator_bridge.cc [hunk 3]
```diff
@@ -23,10 +23,32 @@ void Init(napi_env env, napi_value exports, napi_value module, void* priv) {
...
+  size_t argc = 2;
+  napi_value args[2], sum;
```
Comment (romandev) [inline examples/calculator_bridge.cc]: We should declare one value per one type.
napi_value args[2];
napi_value sum;

File: examples/calculator_bridge.cc [hunk 4]
```diff
@@ -23,10 +23,32 @@ void Init(napi_env env, napi_value exports, napi_value module, void* priv) {
...
+  napi_valuetype valuetype0, valuetype1;
```
Comment (romandev) [inline examples/calculator_bridge.cc]: ditto

**Review by romandev — APPROVED** [review]

**Pull request merged**